src/main/features/core/tray.js

```diff
@@ -15,12 +15,14 @@ let audioDeviceMenu = [
 
 if (process.platform === 'darwin') {
   appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/macTemplate.png`));
+} else if (WindowManager.getWindowManagerGDMName() === 'linux') {
```

Comment:
This is incorrect and won't do what you want.

This `else if` statement should be beneath the `kde-plasma` check and it should be

`process.platform === 'linux'`